### ISSUE TITLE
Karazhan (Black Morass) pre quest 

### ISSUE THREAD

**Issue by gunbilgen** [elysium-project/tbc-bug-tracker#1057]
**Summary:** i've completed "Return to Andormu" and the other parts of chain (OHF escort quest etc.) but NPC didn't give "The Black Morass" quest. So i can not complete Karazhan&Black Morass attune.

**Steps to Reproduce:** please make quest available 

**Expected Results:** i want to be attuned for Karazhan and want to go Black Morass

**Actual Result:**

**Additional Information:**

**Comment by HamonTim** [comment]
Are you 70?
Because "The Black Morass" can only be taken as 70.

**Comment by gunbilgen** [comment]
Of course, when i complete the "Return to Andormu" quest normally Andormu gives "The Black Morass" quest but he only sells arrow for now.
